### ISSUE TITLE
Not working with react@16.x.x due to react-progressbar.js dependency

### ISSUE THREAD

**Issue by gauravdhiman** [Detaysoft/react-chat-elements#53]
Getting below error when using this package with react 16.x.x version. When I drill down, I figured out that one of the dependencies of this package expects the react to be 15.x.x (using `React.createClass()` that is not now available in 16.x.x)

Can you make this package work with React 16.x.x ?

Below is the error we see in browser's developer console when this package is used with React 16.x.x
```js
React.createClass is not a function
TypeError: React.createClass is not a function
    at Object.<anonymous> (http://localhost:3030/_next/-/page/index.js:56379:23)
    at webpackJsonp.../node_modules/react-progressbar.js/dist/react-progressbar.js.Object.assign.i (http://localhost:3030/_next/-/page/index.js:56351:37)
```

**Comment by abdurrahmanekr** [comment]
[This PR](https://github.com/kimmobrunfeldt/react-progressbar.js/pull/26) seems to solve the problem but not yet merged [and has not been developed for a long time.](https://github.com/kimmobrunfeldt/react-progressbar.js/pull/26#issuecomment-367181427)
We can use [react-bar-proccess.js](https://github.com/squarecat/react-progressbar.js) in this case. If you can open a new PR for it, you are very helpful :)

**Comment by abdurrahmanekr** [comment]
This issue resolved with #56
